krod/modules/research/academic_research.py

```
@@ -13,14 +16,30 @@ class AcademicSearch:
     
     def __init__(self, config: Optional[Dict[str, Any]] = None):
         self.config = config or {}
-        self.crossref = Crossref()
-        self.logger = logging.getLogger("krod.academic_search")
+    
+
+        # Get configuration values
+        crossref_email = "research@kroskod.com"
+        user_agent = self.config.get("user_agent", "Krod AI Research Partner (https://krod.kroskod.com)")
```

Flake8 (7.2.0)

[error] local variable 'user_agent' is assigned to but never used

(F841)